### ISSUE TITLE
Create an image gallery with links to the code that generates them

### ISSUE THREAD

**Issue by StevenCHowell** [bokeh/datashader#288]
In reference to issue #286, and building on the idea of a list of examples (issue #231) it can be difficult to find the salient bits of code for applying Datashader to a given problem. In an effort to figure out how to use DataShader for lines, I started going through all of the [notebooks referenced on the README page](https://anaconda.org/jbednar/notebooks). You are probably already planning on this, but it would be great to have a gallery with links to code.  

Ideally these would start with the simplest examples, those that do not involve geographical tiles and preferably using simple Numpy arrays, then progressing to the more fancy examples with DataFrames, taxi, census, flight trajectories.  The 5 gaussian distributions of dots is a great simple example, though a version without the group labels would make this more similar to unsupervised machine learning problems.  I tried to write the sine example used in #286 as an example appropriate to many scientific situations I have seen that would benefit from DataShader (partially in not requiring a downloaded data file, though some added error would make it more meaningful).

**Comment by jbednar** [comment]
Yes, definitely.  We need a good, visual way to organize the examples, and a gallery is my default choice also, with each bit linking to code.  We now have so many examples that it can be very difficult to find the right one as a starting point.
